content/home.md

```diff
@@ -58,123 +58,120 @@ Quire is an ideal tool for anyone seeking to publish a beautiful, scholarly digi
 </div>
 
 </div>
 </div>
-<div class="home-block">
 
+<div class="home-block">
 <div class="home-block__header">
 
-## See Quire in Action
+## Features & Functionality
 
 <div class="more-link">
 
-[Visit Our Community Showcase](/community/community-showcase/)
+[See the Complete Feature List](/about/quire/#features--functionality)
 
 </div>
 </div>
+<div class="home-block__body feature-grid">
 
-{{< q-showcase projects="50_x_50, french_paintings, shifting_perspectives, ray_komai, tilt_art_community" class="full-width-slider" sort="false" >}}
+<div class="image-list">
 
-</div>
+**Flexible Design**
 
-<div class="home-block">
-<div class="home-block__header">
+Allows for a wide range of possible custom features and styles
 
-## The Quire Community
+![document icon](/img/icons/document-center.png)
 
 </div>
-<div class="home-block__body">
 
-Developed by Getty, Quire is now being used by a growing community of scholars, publishers, curators, developers, and the digitally curious. The Quire community is vibrant and diverse, representing a broad spectrum of institutions that provide vital scholarship in the arts and humanities, at all sizes and scales. We support and empower our community at every level of technical experience. All are welcome.
-
-Joining the Quire community is as easy as [becoming a user](https://docs.google.com/forms/d/e/1FAIpQLScKOJEq9ivhwizmdazjuhxBII-s-5SUsnerWmyF8VteeeRBhA/viewform), [participating in the forum](https://github.com/thegetty/quire/discussions), [contributing to code](https://github.com/thegetty/quire/blob/master/CONTRIBUTING.md), or [attending/hosting a Quire event](/community/news-events/), among other exciting opportunities.
+<div class="image-list">
 
-<div class="action-button">
+**Multimedia Support**
 
-[Explore Ways to Get Involved](/community/join-us/)
+Supports visually rich scholarship in the tradition of print publishing
 
-</div>
+![document icon](/img/icons/document-center.png)
 
-</div>
 </div>
 
-<div class="home-block">
-<div class="home-block__header">
+<div class="image-list">
 
-## Key Features & Functionality
+**Single-source Publishing**
 
-<div class="more-link">
+Change content once to change it across all formats, without duplicating effort
 
-[See the Complete Feature List](/about/quire/#features--functionality)
+![document icon](/img/icons/document-center.png)
 
 </div>
-</div>
-<div class="home-block__body feature-grid">
 
 <div class="image-list">
 
-**Multiformat Output**
+**Simplified Structure**
 
-Outputs publications in multiple formats (web, ebook, PDF, print-on-demand)
+Keeps source content in nonspecialized plain text to ensure reusability
 
 ![document icon](/img/icons/document-center.png)
 
 </div>
 
 <div class="image-list">
 
-**Multimedia Support**
+</div>
+</div>
 
-Supports visually rich scholarship in the tradition of print publishing
+<div class="home-block">
 
-![document icon](/img/icons/document-center.png)
+<div class="home-block__header">
+
+## Quire Publications
```

Comment:
I like just calling it "Quire Publications" but I think it should move back up above the Features & Functionality section. It's sill not as high up on the page as we had it originally, and I think it's better to break up the two list-like sections.